DataFormats/L1CaloTrigger/src/classes_def.xml

```diff
@@ -13,4 +13,8 @@
   <class name="std::vector<L1CaloRegion>"/>
   <class name="edm::Wrapper<std::vector<L1CaloEmCand>>" splitLevel="0"/>
   <class name="edm::Wrapper<std::vector<L1CaloRegion>>" splitLevel="0"/>
+
+  <class name="l1t::CICADABxCollection"/>
+  <class name="BXVector<float>"/>
```

Comment:
isn't
```
<class name="BXVector<float>"/>
```
a duplicate of
```
<class name="l1t::CICADABxCollection"/>
```
?

Reply:
Just a misunderstanding, this has been cleaned up.